test/jdk/sun/security/pkcs11/ec/policy

```diff
@@ -2,6 +2,8 @@ grant {
     permission java.lang.RuntimePermission "setSecurityManager";
     permission java.security.SecurityPermission "insertProvider.*";   
     permission java.security.SecurityPermission "removeProvider.*";
+    permission java.security.SecurityPermission
+                       "getProperty.jdk.epkcs8.defaultAlgorithm";
```

Comment:
Maybe we should read this property with `doPrivileged`?

Reply:
I don't see it needing any extra permissions than others.

Reply:
I meant even if you wrap the `Security.getProperty("jdk.epkcs8.defaultAlgorithm")` call in a `doPrivileged` then there is no need to add a new permission here.

Reply:
ok.  I'll switch it